Cargo.toml

```diff
@@ -16,10 +16,15 @@ name = "mask"
 path = "src/lib.rs"
 
 [dependencies]
-clap = "2.33.0"                                                         # https://github.com/clap-rs/clap
+# see https://rust-lang-nursery.github.io/cli-wg/in-depth/machine-communication.html?highlight=atty#whos-reading-this
+atty = "0.2"															# https://github.com/softprops/atty
 colored = "1.8.0"                                                       # https://github.com/mackwic/colored
 pulldown-cmark = { version = "0.5", default-features = false }          # https://github.com/raphlinus/pulldown-cmark
 
+[dependencies.clap]																# https://github.com/clap-rs/clap
+version = "2.33.0"
+features = ["wrap_help", "color"]
```

Comment:
I'll give these a try locally this week. September has turned out to be a very busy month for me 😓